GamepadButtonAndAxisEvents/explainer.md

```diff
@@ -0,0 +1,209 @@
+    const snapshot = changeEvent.gamepadSnapshot;
+
+    for (const axisIndex of changeEvent.axesChanged) {
+      const axisValue = snapshot.axes[axisIndex];
+      console.log(`Axis ${axisIndex} on gamepad ${snapshot.index} changed to ${axisValue}`);
+    }
+
+    // Analog buttons (ex: triggers).
+    for (let buttonIndex of changeEvent.buttonsValueChanged) {
+      const buttonValue = changeEvent.gamepadSnapshot.buttons[buttonIndex].value;
+      console.log('button ' + buttonIndex +
+                  ' on gamepad ' + changeEvent.gamepadSnapshot.index +
+                  ' changed to value ' + buttonValue);
+    }
+
+    // Binary buttons pressed.
+    for (let buttonIndex of changeEvent.buttonsPressed) {
+      const buttonPressed = changeEvent.gamepadSnapshot.buttons[buttonIndex].pressed;
+      console.log('button ' + buttonIndex +
+                  ' on gamepad ' + changeEvent.gamepadSnapshot.index +
+                  ' changed pressed to ' + buttonPressed);
+    }
+
+    // Buttons touched.
+    for (let buttonIndex of changeEvent.buttonsTouched) {
+      const buttonTouched = changeEvent.gamepadSnapshot.buttons[buttonIndex].touched;
+      console.log('button ' + buttonIndex +
+                  ' on gamepad ' + changeEvent.gamepadSnapshot.index +
+                  ' changed touched to ' + buttonTouched);
+    }
+  };
+};
+
+```
+
+### Dependencies on non-stable features
+
+None identified at this stage. The proposal builds upon the existing Gamepad API.
+
+## Alternatives considered
+gamepadchange event: Similar to `rawgamepadinputchange` event but instead the getCoalescedEvents() method is used to return a sequence of events that have been coalesced (combined) together.
+
+How it works: To avoid firing too many events in quick succession for performance issues, the browser may choose to delay firing the gamepadchange event. When this happens, the browser adds the event to an internal queue.
+
+Before running animation callbacks (e.g., requestAnimationFrame), the event queue is flushed. This means that all the events that have been delayed will be combined into one single event, representing the union of all changes up to that point.
+
+The final, combined gamepadchange event represents the combined state of the gamepad from all the events that were delayed and coalesced. When this event is dispatched, it contains all the changes that occurred during the delayed period.
+
+## Accessibility, Privacy, and Security Considerations
+To prevent abuse and fingerprinting, a ["gamepad user gesture"](https://www.w3.org/TR/gamepad/#dfn-gamepad-user-gesture) will be required before `RawGamepadInputChange` events start firing (e.g., pressing a button).
+
+Limit Persistent Tracking (fingerprinting): `rawgamepadinputchange` event will not expose device-specific identifiers. By default, no gamepad state is exposed to the tab even when gamepads are connected. [Fingerprinting in Web](https://www.w3.org/TR/fingerprinting-guidance/)
```

Comment:
nit: can we simplify "Limit Persistent Tracking" by stating that the event will not expose any new state that is not already exposed by polling?